resolwe/flow/tests/test_manager.py

```diff
@@ -124,6 +125,14 @@ def test_dependencies(self):
         data_child1.refresh_from_db()
         data_child2.refresh_from_db()
         data_child3.refresh_from_db()
+
+        # Check locks are created in manager.
```

Comment:
Tests should be in the same commit as the code that they test. Would it make sense to check the state of the access log, i.e. if it is locked or not?